.ghal.rules.json

```diff
@@ -53,6 +53,36 @@
           }
         },
         "technology": {
+          "(?i)csharp-advanced-concepts$": {
+            "labels-add": ":card_file_box: Technology - C# Advanced concepts"
+          },
+          "(?i)csharp-async$": {
+            "labels-add": ":card_file_box: Technology - Async Task"
+          },
+          "(?i)csharp-diagnostics$": {
+            "labels-add": ":card_file_box: Technology - C# / VB diagnostics"
+          },
+          "(?i)csharp-fundamentals$": {
+            "labels-add": ":card_file_box: Technology - C# Fundamentals"
+          },
+          "(?i)csharp-get-started$": {
+            "labels-add": ":card_file_box: Technology - C# Get Started"
+          },
+          "(?i)csharp-language-reference$": {
+            "labels-add": ":card_file_box: Technology - C# Language Reference"
+          },
+          "(?i)csharp-linq$": {
+            "labels-add": ":card_file_box: Technology - LINQ"
+          },
+          "(?i)csharp-null-safety$": {
+            "labels-add": ":card_file_box: Technology - C# Null safety"
```

Comment:
```suggestion
            "labels-add": ":card_file_box: Technology - C# null safety"
```

Reply:
Thanks @pkulikov  eagle :eyes: there.  
I'd proofread this, and missed that one.